Neos.Media/Classes/Domain/Service/ThumbnailService.php

```diff
@@ -108,6 +116,27 @@ public function injectThrowableStorage(ThrowableStorageInterface $throwableStora
      */
     public function getThumbnail(AssetInterface $asset, ThumbnailConfiguration $configuration)
     {
+        // Enforce format conversions if needed. This replaces the actual
+        // thumbnail-configuration with one tha enforces the target format
+        if ($configuration->getFormat() === null) {
+            if ($targetFormat = Arrays::getValueByPath($this->formatConversions, $asset->getMediaType())) {
```

Comment:
the many levels make this a little hard to comprehend. Maybe it makes sense to move the default conversion to a private method and replace this by sth like
```php
if (!$configuration->hasFormat()) {
  $configuration = $this->applyDefaultFormat($configuration);
}
```
?